tests/test_events.py

```diff
@@ -0,0 +1,98 @@
+"""Just here to verify tests are running"""
+
+import mock
+
+from courseware import module_render as render
```

Comment:
Any reason we're not just importing the name unaltered?

Reply:
In the copy/pasted file there was a conflict with a separate module. I'll change it here since there's no conflict